.github/workflows/master.yml

```diff
@@ -9,52 +9,58 @@ on:
       - master
 
 jobs:
-  python35:
+  pip_python35:
+    name: PIP, Python (3.5)
     runs-on: ubuntu-latest
     env:
       SPARK_VERSION: 2.3.4
       PANDAS_VERSION: 0.23.4
       PYARROW_VERSION: 0.10.0
+      # DISPLAY=0.0 does not work in Github Actions with Python 3.5. Here we work around wtih  xvfb-run
```

Comment:
nit: there is two spaces `Here we work around wtih  xvfb-run` -> `Here we work around wtih xvfb-run`